### PR TITLE
feat(components): add post-accordion component

### PR DESCRIPTION
**Pull request by alizedebray**
- [x] Blocked by #2042

### CODE REVIEW

File: packages/documentation/src/stories/components/accordion/accordion.stories.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <span slot="header">Titulum 3</span>
+        <p>Contentus momentus vero siteos et accusam iretea et justo.</p>
+      </post-collapsible>
+    </post-accordion>
+  `
+};
+
+export const CloseOthers: Story = {
+  render: () => html`
+    <post-accordion close-others>
```
Comment (imagoiq) [inline packages/documentation/src/stories/components/accordion/accordion.stories.ts:41]: Nitpick: maybe we should display it similarly to what you wrote in the description, so close-others="true"

File: packages/components/src/components.d.ts [hunk 2]
```diff
@@ -14,6 +14,24 @@ export { HeadingLevel } from "./components/post-collapsible/heading-levels";
 export { BackgroundColor } from "./components/post-tooltip/types";
 export { Placement } from "@floating-ui/dom";
 export namespace Components {
+    interface PostAccordion {
+        /**
+          * If `true`, only one `post-collapsible` can be open at a time.
+         */
+        "closeOthers": boolean;
```
Comment (imagoiq) [inline packages/components/src/components.d.ts:21]: Nitpick: In my opinion, it's something that I would have thing to be the default behavior. I looked at a couple of other similar components and they have rather a `alwaysOpen` prop.

**Review by imagoiq — APPROVED** [review]

File: packages/components/src/components.d.ts (hunk 2 above)
Comment (alizedebray) [reply packages/components/src/components.d.ts:21]: Sure, we can update that. I'll check with the team

Comment (alizedebray) [reply packages/components/src/components.d.ts]: I just changed the default behavior and called the prop `multiple` in https://github.com/swisspost/design-system/pull/2079/commits/326891f3a3cc3d8f6bbdf04b0eff1c35f74630d9.

**Pull request merged**